src/main/java/employee/controller/EmployeeController.java

```diff
@@ -30,8 +36,34 @@
     }
 
     @ResponseBody
-    @RequestMapping(value = "/employees/{id}", method = RequestMethod.GET, produces = "application/json;charset=UTF-8")
-    public ResponseEntity<Employee> getEmployee(@PathVariable("id") String employeeId) {
+    @RequestMapping(value = "/employees/{id}", method = RequestMethod.GET, produces = { "application/vnd.pl.employee+json"})
+    public ResponseEntity getEmployee(@PathVariable("id") String employeeId, HttpServletRequest request) {
+
+        String contentType = request.getHeader("Content-Type");
+        System.out.println(contentType);
+
+        int version = contentType.indexOf("version");
+        System.out.println(contentType.indexOf("version"));
+
+          if(version == -1){
+              return getEmployeeV2(employeeId);
+          }
+          else {
+              String v = contentType.substring(contentType.indexOf("version"));
+
+              if(v.equals("version=1.0")){
```

Comment:
This is not the best way to do this, you want to split out the actual number `1.0` from the `version=` string. Think about using the `String.split()` method

Reply:
done